### PR TITLE
Seeds

### PR DESCRIPTION
**Pull request by anotherkowski**
Created seed data for db using Faker

### CODE REVIEW

File: app/controllers/items_controller.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      flash.now[:alert] = "There was an error saving your item"
+      redirect_to [@user]
+    end
+  end
+
+private
+  def item_params
+    params.require(:item).permit(:name)
+  end
+end
```
Comment (bobbilee19) [inline app/controllers/items_controller.rb:21]: looks good, remove comments and make comments in the PR

File: app/models/user.rb [hunk 2]
```diff
@@ -2,4 +2,31 @@ class User < ActiveRecord::Base
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :confirmable, :registerable, :recoverable, :rememberable, :trackable, :validatable
+
+  # allows a user to have many items associated with account
+  has_many :items, dependent: :destroy
+
+  # validations
```
Comment (bobbilee19) [inline app/models/user.rb]: remove if none

File: app/models/user.rb [hunk 3]
```diff
@@ -2,4 +2,31 @@ class User < ActiveRecord::Base
...
+
```
Comment (bobbilee19) [inline app/models/user.rb]: remove empty spaces here

File: config/routes.rb [hunk 4]
```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
   devise_for :users
 
+  resources :users do
+    resources :items, only: [:create]
+  end
+
+  # resources :items, only: [:create]
```
Comment (bobbilee19) [inline config/routes.rb]: remvoe comments

File: spec/controllers/items_controller_spec.rb [hunk 5]
```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe ItemsController, type: :controller do
+  let(:my_user) { create(:user) }
+  let(:my_item) { create(:item, user: my_user) }
+
+  # Shoulda matchers
```
Comment (bobbilee19) [inline spec/controllers/items_controller_spec.rb]: remove comment

File: spec/controllers/users_controller_spec.rb [hunk 6]
```diff
@@ -0,0 +1,6 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+
```
Comment (bobbilee19) [inline spec/controllers/users_controller_spec.rb]: remove file until using

File: spec/factories/item_factory.rb [hunk 7]
```diff
@@ -0,0 +1,9 @@
+require 'faker'
+
+FactoryGirl.define do
+
```
Comment (bobbilee19) [inline spec/factories/item_factory.rb]: remove space

File: spec/factories/user_factory.rb [hunk 8]
```diff
@@ -0,0 +1,13 @@
+require 'faker'
+
+FactoryGirl.define do
+
```
Comment (bobbilee19) [inline spec/factories/user_factory.rb]: remove space

File: spec/helpers/pages_helper_spec.rb [hunk 9]
```diff
@@ -11,5 +11,5 @@
 #   end
 # end
 RSpec.describe PagesHelper, type: :helper do
-  pending "add some examples to (or delete) #{__FILE__}"
+
```
Comment (bobbilee19) [inline spec/helpers/pages_helper_spec.rb]: remove until using

File: spec/rails_helper.rb [hunk 10]
```diff
@@ -8,12 +8,30 @@
 require 'rspec/rails'
 # note: require 'devise' after require 'rspec/rails'
 require 'devise'
+require 'shoulda/matchers'
+# require 'factory_girl'
```
Comment (bobbilee19) [inline spec/rails_helper.rb]: remove comment

File: spec/spec_helper.rb [hunk 11]
```diff
@@ -96,4 +96,5 @@
   # as the one that triggered the failure.
   Kernel.srand config.seed
 =end
+
```
Comment (bobbilee19) [inline spec/spec_helper.rb:99]: ?

File: spec/views/item/new.html.slim_spec.rb [hunk 12]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "item/new.html.slim", type: :view do
+
```
Comment (bobbilee19) [inline spec/views/item/new.html.slim_spec.rb]: remove file till using

File: spec/views/pages/about.html.slim_spec.rb [hunk 13]
```diff
@@ -1,5 +1,5 @@
 require 'rails_helper'
 
 RSpec.describe "pages/about.html.slim", type: :view do
-  pending "add some examples to (or delete) #{__FILE__}"
+
```
Comment (bobbilee19) [inline spec/views/pages/about.html.slim_spec.rb]: ditto

File: spec/views/pages/home.html.slim_spec.rb [hunk 14]
```diff
@@ -1,5 +1,5 @@
 require 'rails_helper'
 
 RSpec.describe "pages/home.html.slim", type: :view do
-  pending "add some examples to (or delete) #{__FILE__}"
+
```
Comment (bobbilee19) [inline spec/views/pages/home.html.slim_spec.rb]: ditto

File: app/controllers/items_controller.rb (hunk 1 above)
Comment (anotherkowski) [reply app/controllers/items_controller.rb:21]: Edited - done

File: spec/controllers/items_controller_spec.rb (hunk 5 above)
Comment (anotherkowski) [reply spec/controllers/items_controller_spec.rb]: Edited - done

File: spec/factories/user_factory.rb (hunk 8 above)
Comment (anotherkowski) [reply spec/factories/user_factory.rb]: Edited - done

File: spec/helpers/pages_helper_spec.rb (hunk 9 above)
Comment (anotherkowski) [reply spec/helpers/pages_helper_spec.rb]: Okie dokie

File: spec/spec_helper.rb (hunk 11 above)
Comment (anotherkowski) [reply spec/spec_helper.rb:99]: I have no idea why this space is here. Baleted.

File: spec/views/item/new.html.slim_spec.rb (hunk 12 above)
Comment (anotherkowski) [reply spec/views/item/new.html.slim_spec.rb]: Yup, will do

Comment (anotherkowski) [reply spec/views/item/new.html.slim_spec.rb]: done

File: spec/views/pages/about.html.slim_spec.rb (hunk 13 above)
Comment (anotherkowski) [reply spec/views/pages/about.html.slim_spec.rb]: done

File: spec/views/pages/home.html.slim_spec.rb (hunk 14 above)
Comment (anotherkowski) [reply spec/views/pages/home.html.slim_spec.rb]: gone

**Review by anotherkowski — COMMENTED** [review]
Removed unneeded spaces, comments, and empty specs. Resubmitting PR with updates.

**Pull request merged**